elements/lisk-p2p/src/p2p_types.ts

```diff
@@ -62,6 +62,8 @@ export interface P2PNodeInfo {
 	readonly protocolVersion: string;
 	readonly nethash: string;
 	readonly wsPort: number;
+	readonly blockId: string;
```

Comment:
Its too generic, we must make it sepcific for understanding. `lastBlockId`, `finalizedBlockId` or any thing which is specifically been communicated here.

Reply:
Agreed, will change it to `lastBlockId`

Reply:
Done